### PR TITLE
adding eff and calyx mirrors to download page

### PR DESCRIPTION
**Pull request by andpersand**

### CODE REVIEW

File: content/downloading/contents.lr [hunk 1]
```diff
@@ -14,6 +14,10 @@ The safest and simplest way to download Tor Browser is from the official Tor Pro
 
 However, there may be times when you cannot access the Tor Project website: for example, it could be blocked on your network. If this happens, you can use one of the alternative download methods listed below.
 
+##### Mirrors
+
+If you're unable to download Tor Browser from the official Tor Project website, you can instead try downloading it from one of our official mirrors, either through <a href="https://tor.eff.org">EFF</a> or <a href="http://tor.calyxinstitute.org">Calyx Institute</a>.
```
Comment (emmapeel2) [inline content/downloading/contents.lr:19]: "http://tor.calyxinstitute.org" always better to use https when available! ill correct it

**Review by emmapeel2 — APPROVED** [review]

**Comment by emmapeel2** [comment]
This commit has been merged to master, I am not sure why is still here...

**Comment by teor2345** [comment]
This commit is 5aaa34b.

But the merged commit was f18b4d9:

> "http://tor.calyxinstitute.org" always better to use https when available! ill correct it

You can close pull requests manually when their commits don't match exactly.

**Comment by teor2345** [comment]
I think you need write permissions to close pull requests: hiro or I can set that up.

**Comment by emmapeel2** [comment]
> I think you need write permissions to close pull requests: hiro or I can set that up.

I would like to have them on this repo, and support.git too.

**Comment by teor2345** [comment]
> > I think you need write permissions to close pull requests: hiro or I can set that up.
> 
> I would like to have them on this repo, and support.git too.

I opened a trac ticket with a few questions:
https://trac.torproject.org/projects/tor/ticket/28643

Maybe @hiromipaw can answer the questions?

**Pull request closed without merging**